dist/view.js

```diff
@@ -298,4 +299,43 @@ function reactVirtualizedToReactVirtualized(view) {
     rowCount: rowCount,
     columnWidth: columnWidth
   };
+}
+
+/**
+  * Ensure dates and times in resources are in ISO format.
+  * @param {resource}
+  * @return {resource} with formatted dates and times
+  */
+function normalizeDateAndTime(resource) {
+  if (resource._values) {
+    var dateFields = [],
+        timeFields = [],
+        dateTimeFields = [];
+
+    resource.schema.fields.forEach(function (field, idx) {
+      if (field.type === 'date') {
+        dateFields.push(idx);
+      } else if (field.type === 'time') {
+        timeFields.push(idx);
+      } else if (field.type === 'datetime') {
+        dateTimeFields.push(idx);
+      }
+    });
+
+    resource._values.forEach(function (entry) {
+      dateFields.forEach(function (fieldIdx) {
+        var date = entry[fieldIdx].toString();
+        date = new Date(date.substring(0, 28));
+        entry[fieldIdx] = date.toISOString().substring(0, 10);
```

Comment:
So we are first turning Date object into string (line 327), after creating new Date object (line 328) and then turning back into the string?
`entry[fieldIdx].toISOString().substring(0, 10)` would not work?